jira-mixin/alerts/general.yaml

```
@@ -0,0 +1,43 @@
+groups:
+- name: alert.rules
+  rules:
+  - alert: InstanceDown
+    expr: up == 0
+    for: 1m
+    labels:
+      severity: "critical"
+    annotations:
+      summary: "JIRA server down"
+      description: "The JIRA server has been down for more than 1 minute."
+  - alert: LicenseExpired
+    expr: jira_maintenance_expiry_days_gauge == 0
+    for: 1m
+    labels:
+      severity: "critical"
+    annotations:
+      summary: "JIRA license expired"
+      description: "The JIRA license has expired."
+  - alert: LicenseWarning
+    expr: jira_maintenance_expiry_days_gauge == 7

```

I think this'll stop firing once it goes down to 6 days, so probably should use a `<=` here 😉

```suggestion
    expr: jira_maintenance_expiry_days_gauge <= 7
```

---

Good point. 

I updated it to be:
`jira_maintenance_expiry_days_gauge <= 7 and jira_maintenance_expiry_days_gauge > 0`

so that it doesn't overlap with the expired alert.